assets/js/modules/analytics-4/datastore/webdatastreams.js

```diff
@@ -322,6 +322,50 @@ const baseSelectors = {
 	getWebDataStreamsBatch( state, propertyIDs ) {
 		return pick( state.webdatastreams, propertyIDs );
 	},
+
+	/**
+	 * Gets the matched measurement IDs for the given properties.
+	 *
+	 * @since 1.88.0
+	 *
+	 * @param {Object}         state      Data store's state.
+	 * @param {Array.<Object>} properties GA4 properties array of objects.
+	 * @return {(Object|null)} Matched property ID as the key and measurement ID as the value; otherwise, an empty object. Null if no properties are provided.
```

Comment:
The description could be slightly clarified -
```suggestion
	 * @return {(Object|null)} Object with matched property ID as the key and measurement ID as the value, or an empty object if no matches are found. Null if no properties are provided.
```